scripts/install.sh

```diff
@@ -188,8 +322,8 @@ if [ "$DISABLE_NETWORKING" = "true" ]; then
   sed -i "s/photonvision.jar/photonvision.jar -n/" /lib/systemd/system/photonvision.service
 fi
 
-if [[ -n $(cat /proc/cpuinfo | grep "RK3588") ]]; then
-  echo "This has a Rockchip RK3588, enabling all cores"
+if grep -q "RK3588" /proc/cpuinfo; then
+  debug "This has a Rockchip RK3588, enabling all cores"
   sed -i 's/# AllowedCPUs=4-7/AllowedCPUs=0-7/g' /lib/systemd/system/photonvision.service
```

Comment:
I made a mistake in an earlier commit on this line. It should only enable CPUs 4-7:
```suggestion
  sed -i 's/# AllowedCPUs=4-7/AllowedCPUs=4-7/g' /lib/systemd/system/photonvision.service
```